pulse/pacat-simple-vchan.c

```diff
@@ -210,6 +215,16 @@ static void process_playback_data(struct userdata *u, pa_stream *s, size_t max_l
         return;
     }
 
+    if (verbose > 1) {
+        pa_usec_t latency = 0;
+        int negative;
```

Comment:
```suggestion
        int negative = 0;
```

avoid logging junk from the stack

Reply:
I don't use the value at all, so it doesn't matter.